src/definitions.jl

```diff
@@ -271,7 +271,14 @@ summary(p::ScaledPlan) = string(p.scale, " * ", summary(p.p))
 *(p::Plan, I::UniformScaling) = ScaledPlan(p, I.λ)
 
 # Normalization for ifft, given unscaled bfft, is 1/prod(dimensions)
-normalization(::Type{T}, sz, region) where T = one(T) / Int(prod(sz[r] for r in region))::Int
+# ensure that region is a subset of eachindex(sz).
+_checkindex(sz, region::AbstractVector) = checkindex(Bool, sz, region)
+# this method handles the case where region is not an array, e.g. it is a Tuple
+_checkindex(sz, region) = all(r -> checkindex(Bool, eachindex(sz), r), region)
```

Comment:
Isn't the caller already calling `eachindex`?

Reply:
Indeed, fixed now